File: app/src/main/java/com/brainwellnessspa/BillingOrderModule/Models/CardListModel.java
package com.brainwellnessspa.BillingOrderModule.Models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CardListModel {
    @SerializedName("ResponseData")
    @Expose
    private List<ResponseData> responseData = null;
    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseMessage")
    @Expose
    private String responseMessage;
    @SerializedName("ResponseStatus")
    @Expose
    private String responseStatus;

    public List<ResponseData> getResponseData() {
        return responseData;
    }

    public void setResponseData(List<ResponseData> responseData) {
        this.responseData = responseData;
    }

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseMessage() {
        return responseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        this.responseMessage = responseMessage;
    }

    public String getResponseStatus() {
        return responseStatus;
    }

    public void setResponseStatus(String responseStatus) {
        this.responseStatus = responseStatus;
    }

    public class ResponseData {
        @SerializedName("isDefault")
        @Expose
        private String isDefault;
        @SerializedName("brand")
        @Expose
        private String brand;
        @SerializedName("image")
        @Expose
        private String image;
        @SerializedName("customer")
        @Expose
        private String customer;
        @SerializedName("exp_month")
        @Expose
        private String expMonth;
        @SerializedName("exp_year")
        @Expose
        private String expYear;
        @SerializedName("last4")
        @Expose
        private String last4;

        public String getIsDefault() {
            return isDefault;
        }

        public void setIsDefault(String isDefault) {
            this.isDefault = isDefault;
        }

        public String getImage() {
            return image;
        }

        public void setImage(String image) {
            this.image = image;
        }

        public String getBrand() {
            return brand;
        }

        public void setBrand(String brand) {
            this.brand = brand;
        }

        public String getCustomer() {
            return customer;
        }

        public void setCustomer(String customer) {
            this.customer = customer;
        }

        public String getExpMonth() {
            return expMonth;
        }

        public void setExpMonth(String expMonth) {
            this.expMonth = expMonth;
        }

        public String getExpYear() {
            return expYear;
        }

        public void setExpYear(String expYear) {
            this.expYear = expYear;
        }

        public String getLast4() {
            return last4;
        }

        public void setLast4(String last4) {
            this.last4 = last4;
        }
    }
}

File: app/src/main/java/com/brainwellnessspa/billingOrderModule/models/CardListModel.kt
package com.brainwellnessspa.billingOrderModule.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CardListModel {
    @SerializedName("ResponseData")
    @Expose
    var responseData: List<ResponseData>? = null

    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = null

    @SerializedName("ResponseMessage")
    @Expose
    var responseMessage: String? = null

    @SerializedName("ResponseStatus")
    @Expose
    var responseStatus: String? = null

    inner class ResponseData {
        @SerializedName("isDefault")
        @Expose
        var isDefault: String? = null

        @SerializedName("brand")
        @Expose
        var brand: String? = null

        @SerializedName("image")
        @Expose
        var image: String? = null

        @SerializedName("customer")
        @Expose
        var customer: String? = null

        @SerializedName("exp_month")
        @Expose
        var expMonth: String? = null

        @SerializedName("exp_year")
        @Expose
        var expYear: String? = null

        @SerializedName("last4")
        @Expose
        var last4: String? = null
    }
}